### ISSUE TITLE
Unclear "catch (ApplicationException ex)"

### ISSUE THREAD

**Issue by jinek** [dotnet/docs#6258]
Why was it put: "catch (ApplicationException ex)
    {
        writer.WriteLine("ApplicationException Message: {0}", ex.Message);
    }
" and in case there is a reason, why rest of the code was not put in to this try/catch

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 85197d9e-6a02-c46b-5f1a-a7970bea5e06
* Version Independent ID: b63d527f-ee65-11f4-cc5b-bfae96df87a9
* Content: [Implementing an Implicit Transaction using Transaction Scope](https://docs.microsoft.com/en-us/dotnet/framework/data/transactions/implementing-an-implicit-transaction-using-transaction-scope?view=netframework-4.7)
* Content Source: [docs/framework/data/transactions/implementing-an-implicit-transaction-using-transaction-scope.md](https://github.com/dotnet/docs/blob/master/docs/framework/data/transactions/implementing-an-implicit-transaction-using-transaction-scope.md)
* Product: **dotnet-framework**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by rpetrusha** [comment]
Thanks for pointing this out, @jinek. This is probably a legacy code example that was written before the guidance that ApplicationException not be caught was issued. I'm especially thankful for your opening this issue, since it led me back to #4870, where I wrote that I'd provide you with guidance for getting started. I failed to do that, and I apologize. I'll add some comments to the original issue on what to do. If you'd still like to work on it, we'd really appreciate your help.
In the meantime, I'll close this issue, since the duplicated ApplicationException is covered by #4870.
